### PR TITLE
Decompiled and Matched unknown_move

### PR DESCRIPTION
**Pull request by Nasina7**
Decompiled battle/move/unknown_move

### CODE REVIEW

File: src/battle/move/unknown_move/779C90.c [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+
+    script->varTable[0] = 99;
+
+    enemyTargetPowerBounceChance = enemyTargetActor->actorBlueprint->powerBounceChance;
+
+    if (is_ability_active(0) != 0) {
+        enemyTargetPowerBounceChance += 7;
+    }
+
+    D_802A26D0_77B360 *= enemyTargetPowerBounceChance;
```
Comment (pixel-stuck) [inline src/battle/move/unknown_move/779C90.c:25]: Is the / 100 required on a separate line, or could the entire calculation be on this line only?

File: src/battle/move/unknown_move/779C90.c [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
 #include "world/common/UnkMoveFunc1.inc.c"
 
-INCLUDE_ASM(s32, "battle/move/unknown_move/779C90", func_802A10E4_779D74);
+extern s32 D_802A26D0_77B360;
+
+ApiStatus func_802A10E4_779D74(Evt* script, s32 isInitialCall) {
+    Actor *enemyTargetActor;
+    u32 enemyTargetPowerBounceChance;
+
+    enemyTargetActor = get_actor((s32) get_actor(script->owner1.enemyID)->targetActorID);
```
Comment (pixel-stuck) [inline src/battle/move/unknown_move/779C90.c:15]: Is the s32 cast needed here?

File: src/battle/move/unknown_move/779C90.c (hunk 1 above)
Comment (pixel-stuck) [reply src/battle/move/unknown_move/779C90.c:25]: Also, I guess this global would be chance as percentage, though maybe it's used elsewhere

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.00%) on `us`.

File: src/battle/move/unknown_move/779C90.c (hunk 2 above)
Comment (Nasina7) [reply src/battle/move/unknown_move/779C90.c:15]: Nope, just removed it :+1:

File: src/battle/move/unknown_move/779C90.c (hunk 1 above)
Comment (Nasina7) [reply src/battle/move/unknown_move/779C90.c:25]: It seems like it does need to be on a separate line, although I could just be missing something.

Comment (pixel-stuck) [reply src/battle/move/unknown_move/779C90.c:25]: Globals are a bit weird, so it probably is correct as you have it, but I figured it was worth a shot :)

Comment (Nasina7) [reply src/battle/move/unknown_move/779C90.c:25]: :+1:

File: src/battle/move/unknown_move/779C90.c [hunk 3]
```diff
@@ -1,7 +1,34 @@
...
+    Actor *enemyTargetActor;
+    u32 enemyTargetPowerBounceChance;
+
+    enemyTargetActor = get_actor(get_actor(script->owner1.enemyID)->targetActorID);
+
+    script->varTable[0] = 99;
+
+    enemyTargetPowerBounceChance = enemyTargetActor->actorBlueprint->powerBounceChance;
+
+    if (is_ability_active(0) != 0) {
```
Comment (nanaian) [inline src/battle/move/unknown_move/779C90.c:21]:
```suggestion
    if (is_ability_active(ABILITY_DODGE_MASTER)) {
```

**Review by nanaian — CHANGES_REQUESTED** [review]

File: src/battle/move/unknown_move/779C90.c [hunk 4]
```diff
@@ -1,7 +1,34 @@
...
+
 #define NAMESPACE battle_move_unknown_move
 
 #include "world/common/UnkMoveFunc1.inc.c"
 
-INCLUDE_ASM(s32, "battle/move/unknown_move/779C90", func_802A10E4_779D74);
+extern s32 D_802A26D0_77B360;
+
+ApiStatus func_802A10E4_779D74(Evt* script, s32 isInitialCall) {
+    Actor *enemyTargetActor;
```
Comment (ethteck) [inline src/battle/move/unknown_move/779C90.c:12]: Pointer on the left pleez (Actor*) and also please move the enemyTargetActor initialization into this line as well

**Review by ethteck — CHANGES_REQUESTED** [review]

**Pull request merged**